src/components/header/header.html

```diff
@@ -20,7 +20,10 @@ <h3 class="docs-block__header">Example usage</h3>
         <div class="mint-header">
             <div class="mint-header__container">
                 <div class="mint-header__left">
-                    <a href="#" class="mint-home-button"></a>
+                    <a href="#" class="mint-home-button">
```

Comment:
We (me and @kdzwinel ) have decided that for now this is the best solution. (even though it will download both images - as both images will be used on mobile only drawback will be desktop downloading mobile image)